wordpress-mobile/WordPress-Android
# Feature/activity log coroutines

This PR adds coroutines to Activity log. Most of the changes to the production code are relatively simple. The main change is going from 
```
dispatcher.dispatch(FetchAction(FetchPayload()))
@Subscribe
fun onFetched(event: OnFetch){}
```
to the coroutine simplification
```
launch {
val onFetch = store.fetchAction(FetchPayload())
onFetched(onFetch)
}
```
This inlines the async call and makes the code more readable.
To test:
- go to activity log
- load next page
- go to a detail
- go back
- click on rewind on an activity
- you see a rewind item on the first position
- rewind ends successfuly
